plugins/flatpak/gs-flatpak.c

```diff
@@ -3555,13 +3555,24 @@ gs_flatpak_app_copy (GsFlatpak *self,
 						     app_arch, app_branch);
 	const gchar* app_collection_id = gs_flatpak_get_collection_id (self,
 								       app);
-
+	g_autofree gchar* installation_arg = NULL;
 	gboolean spawn_retval;
-	const gchar *argv[] = {"/usr/bin/flatpak", "--system", "create-usb",
-			       copy_dest, app_ref, NULL};
+	const gchar *argv[6];
```

Comment:
Nitpick: Maybe the code would be more readable and easier to change if you assign the array here too.
```
... argv[] = {"/usr/bin/flatpak",
              NULL, /* installation arg */
              "create-usb",
              ...,
              NULL
             };
(... stuff)
argv[1] = installation_arg;
```

Reply:
Pushed a fixup